### PR TITLE
Bsp 134 section 5

### PR DESCRIPTION
**Pull request by jakub-kozlowski**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/functionalTests/java/uk/gov/hmcts/reform/finrem/functional/documents/DocumentValidationTests.java [hunk 1]
```diff
@@ -69,7 +70,7 @@ public void verifyDocumentForConsentOrderShouldReturnOkResponseCode() throws Exc
         Response response = utils.getResponseData(responseToOrderFileCheckUrl, callbackRequest);
         int statusCode = response.getStatusCode();
```
Comment (jvictorcf) [inline src/functionalTests/java/uk/gov/hmcts/reform/finrem/functional/documents/DocumentValidationTests.java]: Minor comment. I'd the StatusCode and compare it with HttpStatus.OK,

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java [hunk 2]
```diff
@@ -54,9 +56,33 @@
...
+    }
+    
+    private void mapAuthorisationSignedToYesOrNo(String ocrFieldName, String ccdFieldName,
+                                                 List<OcrDataField> ocrDataFields, Map<String, Object> formSpecificMap) {
+        ocrDataFields.stream()
+            .filter(ocrDataField -> ocrDataField.getName().equals(ocrFieldName))
+            .map(OcrDataField::getValue)
+            .findFirst()
+            .ifPresent(ocrValue -> {
+                String ccdValue = ocrValue.trim().isEmpty() ? "No" : "Yes";
```
Comment (jvictorcf) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java]: We have constants for this (at least in Divorce).

Comment (yarpo) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java]:
```
uk.gov.hmcts.reform.finrem.caseorchestration.OrchestrationConstants.YES_VALUE
uk.gov.hmcts.reform.finrem.caseorchestration.OrchestrationConstants.NO_VALUE
```

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/validation/FormAValidator.java [hunk 3]
```diff
@@ -61,9 +66,13 @@
...
         DIVORCE_STAGE_REACHED,
-        APPLICANT_REPRESENTED
+        APPLICANT_REPRESENTED,
+        AUTHORISATION_NAME,
+        AUTHORISATION_SIGNED,
+        AUTHORISATION_SIGNED_BY,
+        AUTHORISATION_DATE
     );
-
+
```
Comment (yarpo) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/validation/FormAValidator.java]: not needed :)

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java [hunk 4]
```diff
@@ -260,6 +274,31 @@ public void shouldReplaceOrderForChildrenNoAgreementReasons() {
                 + "absent parent of the child is not habitually resident in the United Kingdom",
             "When not habitually resident");
     }
+    
+    @Test
+    public void shouldTransformEmptyAuthorisationSignedToNo() {
+        assertOnSingleFieldTransformationResult(
+            OcrFieldName.AUTHORISATION_SIGNED, "",
+            "authorisationSigned", "No");
```
Comment (yarpo) [inline src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java]: const

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java [hunk 5]
```diff
@@ -260,6 +274,31 @@ public void shouldReplaceOrderForChildrenNoAgreementReasons() {
...
+    }
+
+    @Test
+    public void shouldTransformAnyAuthorisationSignedToYes() {
+        assertOnSingleFieldTransformationResult(
+            OcrFieldName.AUTHORISATION_SIGNED, "Any non-empty value should become yes",
+            "authorisationSigned", "Yes");
```
Comment (yarpo) [inline src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java]: const

**Pull request merged**